solutions/usaco-1179.mdx

```diff
@@ -0,0 +1,90 @@
+---
+id: usaco-1179
+source: USACO Gold 2022 January
+title: Herdle
+author: Juheon Rhee
+---
+
+[Official Editorial (Java)](http://www.usaco.org/current/data/sol_prob1_bronze_jan22.html)
+
+## Implementation
+
+**Time Complexity:** $\mathcal{O}(N^2)$
+
+<LanguageSection>
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+
+int main(){
+
+	char correct[9], guess[9];
+
+	for(int i = 0; i < 9; i++){
+		cin >> correct[i];
+	}
+	
+	for(int i = 0; i < 9; i++){
+		cin >> guess[i];
+	}
+
+	int green = 0, yellow = 0;
+
+	for(int i = 0; i < 9; i++){
```

Comment:
spaces for control flow statements
like
```suggestion
	for (int i = 0; i < 9; i++) {
```

Reply:
do this for the others as well

Reply:
okay

Reply:
bump